bugwarrior/services/gitlab.py

```diff
@@ -433,7 +431,21 @@ def include_todo(todo):
 
     def issues(self):
         tmpl = '{scheme}://{host}/api/v4/projects'
-        all_repos = self._fetch_paged(tmpl)
+
+        all_repos = []
+        if self.include_repos:
+            for repo in self.include_repos:
+                indiv_tmpl = tmpl + '/' + quote(repo, '')
+                item = self._fetch(indiv_tmpl)
+                if not item:
+                    break
+
+                all_repos += [ item ]
+
+        if (self.include_regex is not None
+            or not self.include_repos):
```

Comment:
I think this should just be an `else` statement (per above change recommendations).

Reply:
Updated in 849494e